pkg/volume/csi/csi_mounter.go

```diff
@@ -199,7 +205,8 @@ func (c *csiMountMgr) SetUpAt(dir string, mounterArgs volume.MounterArgs) error
 			nodeName := string(c.plugin.host.GetNodeName())
 			c.publishContext, err = c.plugin.getPublishContext(c.k8s, volumeHandle, string(driverName), nodeName)
 			if err != nil {
-				return err
+				// we could have a transient error associated with fetching publish context
+				return volumetypes.NewTransientOperationFailure(err.Error())
```

Comment:
why this one does not use fmt.Sprintf("") to generate errors as below?

Reply:
I think mostly because this error here does not wrap existing error into a string. We can do that here though..